lxdock/provisioners/ansible.py

```diff
@@ -16,13 +16,13 @@ class AnsibleProvisioner(Provisioner):
     name = 'ansible'
 
     guest_required_packages_alpine = ['openssh', 'python', ]
-    guest_required_packages_archlinux = ['openssh', 'python2', ]
+    guest_required_packages_arch = ['openssh', 'python', ]
```

Comment:
@hsoft I didn't realize the Python versions support... python2 was installed as `python2` binary so I got an error of "Cannot find python". Perhaps I need a symlink, or if there's an Ansible configuration on which Python to use, I can also do it.